recipes/ruby/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "ruby" %}
+{% set major_minor = "2.4" %}
+{% set version = "2.4.1" %}
+{% set sha256 = "a330e10d5cb5e53b3a0078326c5731888bb55e32c4abfeb27d9e7f8e5d000250" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://cache.ruby-lang.org/pub/ruby/{{ major_minor }}/ruby-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain3
+    - make
+    - cmake
+    - openssl 1.0.*
+    - gcc  # [osx]
```

Comment:
Why is `gcc` still needed on macOS? I would expect now that the `gmp` requirement is satisfied that `gcc` won't be needed.